### DISCUSSION TITLE
A single DU with multiple RUs

### DISCUSSION THREAD

**Discussion by yc541** [srsran/srsRAN_Project#840] [General Help]
Just out of curiosity, is it possible for the DU to connect to multiple 7.2 split RUs? For split 8 it is quite straightforward and I have successfully got 2 USRP X310 (with external clock/pps) connected to a single DU as a 4T4R gNB. BY looking at the DU code, it sees 4 antennas (from a multi_usrp object) rather than 2 separate RUs. 

I wonder if it would be similar when using a 7.2 split RU? Will the second RU not connect to the DU (if there is already an RU connected to it)? I'd love to test something but I don't have a proper 7.2 split RU.

**Comment by andrepuschmann** [comment]
Hey @yc541 - yes, this is possible. If you're using the gNB and configure two cells/sectors it'll internally instantiate two (or more) DU objects each serving one ORU.

**Reply by yc541** [reply]
@andrepuschmann Many thanks for the reply. Apologies for not describing the scenario clearly, I was trying to find out if it is possible to  deploy a gNB with a single cell/single DU but with multiple radios. What I did with 2 USRP X310 (2T2R) was configuring a gNB with a single cell with 4T4R antennas, which connected the single cell/DU to 2 USRPs (this deployment was successful). 

Would it be possible to replicate this deployment with two 7.2 split RUs? I don't have any ORUs to test this. I know SRS does not support beyond 4T4R, just wonder if the OFH can physically stream the IQs from two ORUs to the PHY of a single DU.
